erde/cli/repl.lua

```diff
@@ -45,13 +50,14 @@ local function repl()
       -- Try input as an expression first! This way we can still print the value
       -- in the case that the expression is also a valid block (i.e. function calls).
       ok, result = pcall(function()
-        return lib.run('return ' .. source, { alias = 'stdin' })
+        -- pack results so we know how many were actually returned even if there are nils among them
+        return pack(lib.run('return ' .. source, { alias = 'stdin' }))
       end)
 
       if not ok and type(result) == 'string' and not result:find('unexpected eof') then
         -- Try input as a block
         ok, result = pcall(function()
-          lib.run(source, { alias = 'stdin' })
+          return pack(lib.run(source, { alias = 'stdin' }))
```

Comment:
added a small fix to also print out returns from blocks

Reply:
oops this is originally due to the annoying table print from `_MODULE`. let me get a workaround for this

Reply:
above should be fixed by https://github.com/erde-lang/erde/pull/16/commits/ad27a07b9c1ba9f1c5175ae0dff2329fcb4f52a8